package.json

```diff
@@ -38,13 +38,16 @@
     "test": "jest"
   },
   "lint-staged": {
+    "**/lib/**/*.js": [
+      "./scripts/lint-staged-jest.sh"
+    ],
     "**/package.json": [
       "yarn format:package",
       "git add"
     ],
     "*.js": [
       "prettier --write",
-      "eslint",
+      "./scripts/lint-staged-eslint.sh",
```

Comment:
What is the change here? Reading the script code, it seems it is linting all files for each file that changed. I have the same concern about the change above for the tests. My understanding for how `lint-staged` works is that for each changed file that matches the key in the configuration, the array of scripts is run. In this case, the script here would run 1 time per `.js` file. Am I misunderstanding?

Reply:
@camacho My bad, it was me miss understanding lint-staged. Have retested and revert the changes. Good call.